common_toolset/annotation.yml

```diff
@@ -2,19 +2,5 @@ tool_panel_section_label: Annotation
 tools:
 - name: prokka
   owner: crs4
-- name: busco
-  owner: iuc
-- name: mlst
-  owner: iuc
-- name: abricate
```

Comment:
I'd keep abricate. It gets a lot of use on our servers.

Reply:
ok